src/citrine/informatics/workflows/design_workflow.py

```diff
@@ -47,12 +47,17 @@ class DesignWorkflow(Resource['DesignWorkflow'], Workflow):
     )
     archived = properties.Boolean('archived', default=False)
     created_by = properties.Optional(properties.UUID, 'created_by', serializable=False)
+    updated_by = properties.Optional(properties.UUID, 'updated_by', serializable=False)
+    archived_by = properties.Optional(properties.UUID, 'archived_by', serializable=False)
     create_time = properties.Optional(properties.Datetime, 'create_time', serializable=False)
+    update_time = properties.Optional(properties.Datetime, 'update_time', serializable=False)
+    archive_time = properties.Optional(properties.Datetime, 'archive_time', serializable=False)
     module_type = properties.String('module_type', default='DESIGN_WORKFLOW')
     typ = properties.String('type', default='DesignWorkflow', deserializable=False)
 
     status_description = properties.String('status_description', serializable=False)
     name = properties.String('name')
+    description = properties.Optional(properties.String, 'description')
```

Comment:
ah, i should probably add this to the init too

Reply:
This is probably due to a bad review on my part :-(